src/main/java/stringCalculator/Operator.java

```diff
@@ -0,0 +1,26 @@
+package stringCalculator;
+
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (first, second) -> first + second),
+    MINUS("-", (first, second) -> first - second),
+    MULTIPLY("*", (first, second) -> first * second),
+    DIVIDE("/", (first, second) -> first / second);
+
+    private String symbol;
+    private BiFunction<Integer, Integer, Integer> operation;
+
+    Operator(String symbol, BiFunction<Integer, Integer, Integer> operation) {
+        this.symbol = symbol;
+        this.operation = operation;
+    }
+
+    public String getSymbol(){ return symbol; }
```

Comment:
```suggestion
	public static Operator findSymbol(String input) {
		return Arrays.stream(values())
			.filter(operator -> operator.getSymbol().equals(input))
			.findAny()
			.orElseThrow(() -> new IllegalArgumentException("연산자가 아닙니다."));
	}
```

현재 Operator 에 정의된 내용을 이 Operator를 사용하는 클라이언트 클래스에서 가져다가 로직을 만드는게 아니라
내부적으로 연산자를 찾아서 반환하는 코드를 작성하면 밖에서 반복문 및 조건문을 더 줄일 수 있을 것 같네요.!